### ISSUE TITLE
Java 7 compilation error

### ISSUE THREAD

**Issue by slawekjaranowski** [PayU-Tech/Ratel#32]
When use java 7 to compile we have:

````
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] .../Ratel/core/src/main/java/com/payu/ratel/context/ProcessContext.java:[4,26] package java.util.function does not exist
[ERROR] .../Ratel/core/src/main/java/com/payu/ratel/context/ProcessContext.java:[12,93] cannot find symbol
  symbol:   class Supplier
  location: class com.payu.ratel.context.ProcessContext
[INFO] 2 errors
````
It is some inconsistency because in project compilation is set to target 1.7

**Comment by lukaszPulawski** [comment]
Resolved in pull request #33

**Comment by lukaszPulawski** [comment]
Resolved
